sessions/js-advanced-practice/cards-from-form/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+		<title>HTML & CSS & JS</title>
+		<link rel="stylesheet" href="./css/styles.css" />
+	</head>
+	<body>
+		<!--
+Notice: the written and/or generated html is NOT accessible to keep the code simple. You find an accessible version here: https://codepen.io/neuefische/pen/XWzKoyp.
+-->
+		<form data-js="form" autocomplete="off">
+			<label for="question-input">Question:</label>
+			<textarea
+				required
+				name="question"
+				id="question-input"
+				placeholder="Why ...?"
+			></textarea>
+
+			<label for="answer-input">Answer:</label>
+			<textarea
+				required
+				name="answer"
+				id="answer-input"
+				placeholder="Because ...!"
+			></textarea>
+
+			<label for="tags-input">Tags:</label>
+			<input name="tags" id="tags-input" placeholder="html, css, more..." />
+
+			<button>Create Card</button>
+		</form>
+
+		<ul role="list" class="card-list" data-js="cards"></ul>
+	</body>
+	<script src="./js/index.js"></script>
```

Comment:
```suggestion
		<link rel="stylesheet" href="./css/styles.css" />
		<script src="./js/index.js" defer></script>
	</head>
	<body>
		<!--
Notice: the written and/or generated html is NOT accessible to keep the code simple. You find an accessible version here: https://codepen.io/neuefische/pen/XWzKoyp.
-->
		<form data-js="form" autocomplete="off">
			<label for="question-input">Question:</label>
			<textarea
				required
				name="question"
				id="question-input"
				placeholder="Why ...?"
			></textarea>

			<label for="answer-input">Answer:</label>
			<textarea
				required
				name="answer"
				id="answer-input"
				placeholder="Because ...!"
			></textarea>

			<label for="tags-input">Tags:</label>
			<input name="tags" id="tags-input" placeholder="html, css, more..." />

			<button>Create Card</button>
		</form>

		<ul role="list" class="card-list" data-js="cards"></ul>
	</body>
```